xml/pop_banners.xml

```diff
@@ -217,7 +217,58 @@ matched against these patterns to fingerprint POP3 servers.
     <param pos="0" name="service.product" value="Courier POP"/>
   </fingerprint>
   <!--
-
+<fingerprint pattern= "^CMailServer ([\d\.]+) POP3 Service Ready$">
+    <description>matches to CMailServer</description>
+    <example>CMailServer 5.0.0 POP3 Service Ready</example>
+	<param pos="0" name="service.product" value="CMailServer"/>
+	<param pos="0" name="service.vendor" value="Youngzsoft"/>
+	<param pos="0" name="os.family" value="Windows" />
+	<param pos="0" name="os.vendor" value="Microsoft"/>
+	<param pos= "1" name="service.version"/>
+  </fingerprint>
+<fingerprint pattern= "^POP3 Bigfoot v(\d\.\d) server ready$">
+    <description>matches to POP3 Bigfoot server</description>
+    <example>POP3 Bigfoot v1.0 server ready</example>
+    <param pos="0" name="service.vendor" value="Bigfoot"/>
+    <param pos="0" name="service.family" value="Bigfoot Email Tools"/>
+    <param pos="0" name="service.product" value="Bigfoot Email Tools"/>
+	<param pos="1" name="service.version"/>
+  </fingerprint>
+<fingerprint pattern="^(CCProxy) (\d\.\d) POP3 Service Ready$">
```

Comment:
The parens aren't required in the regex above. Also, you can make a slight change and match all versions of the banner that we've seen.

All the banners that I know about:
```
grep -i 'CCproxy' ./version_recog_unmatched_count.txt 
2182 CCProxy 8.0 POP3 Service Ready
1693 CCProxy 6.3 POP3 Service Ready
 319 CCProxy 2010 POP3 Service Ready
  52 CCProxy 7.2 POP3 Service Ready
  24 CCProxy 7.3 POP3 Service Ready
   7 CCProxy 6.64 POP3 Service Ready
   7 CCProxy 6.6 POP3 Service Ready
   5 CCProxy 6.4.1 POP3 Service Ready
   3 CCProxy 7.0 POP3 Service Ready
   1 CCProxy 6.63 POP3 Service Ready
   1 CCProxy 6.61 POP3 Service Ready
   1 CCProxy 6.5 POP3 Service Ready
   1 CCProxy 6.4.2 POP3 Service Ready
   1 CCProxy 6.4.0 POP3 Service Ready
   1 CCProxy 6.2 POP3 Service Ready
   1 CCProxy 6.0 POP3 Service Ready
```

Regex to match them all:
`^CCProxy ([\d.]+) POP3 Service Ready$`

Typically we'd add an example for each variant of the banner string.  Here's two to get you started:

```
<example service.version="8.0">CCProxy 8.0 POP3 Service Ready</example>
<example service.version="2010">CCProxy 2010 POP3 Service Ready</example>
```